secator/template.py

```
@@ -55,85 +54,192 @@
+				conf['required'] = False
+				conf['default'] = default_from_config
+				conf['default_from'] = node.id
+				if node_opts.get(k):
+					conf['default_from'] = node.id
+				elif ancestor_opts_defaults.get(k):
+					conf['default_from'] = node.ancestor.id
+					conf['prefix'] = 'Config overrides'
+				elif config_opts_defaults.get(k):
+					conf['default_from'] = config.name
+					conf['prefix'] = 'Config overrides'
+				mapped_value = cls.opt_value_map.get(opt_name)
```

Pylint (3.3.7)

[warning] Bad indentation. Found 4 spaces, expected 16

(W0311)